schema/site.schema.json

```
@@ -1935,7 +1935,7 @@
           "maximum": 10000
         },
         "repos": {
-          "description": "List of repositories whose git operations should be recorded.",
+          "description": "List of repositories whose git operations should be recorded. To record commands on all repositories, simply pass in a wildcard as the only item in the array.",
```

I'd phrase it this way to remove possible ambiguity between `*` and `.*`?
```suggestion
          "description": "List of repositories whose git operations should be recorded. To record commands on all repositories, simply pass in an asterisk as the only item in the array.",
```